src/Mumble.proto

```diff
@@ -18,6 +18,9 @@ message Version {
 	optional string os = 3;
 	// Client OS version.
 	optional string os_version = 4;
+	// New version number format.
+	// Necessary since patch level may exceed 255. (See #5827)
```

Comment:
In the comment, we should use a full URL and not just #xxxx as those only link on GitHub.
Or is this actually a full URL and GitHub just shortens this to #xxxx?